obspy/io/mseed/scripts/recordanalyzer.py

```diff
@@ -175,17 +175,22 @@ def _read_fixed_header(self):
             raise
         # Write values to dictionary.
         self.fixed_header['Sequence number'] = \
-            int(''.join(x.decode('ascii') for x in header_item[:6]))
+            int(''.join(x.decode('ascii', errors="warn")
```

Comment:
Does `.decode(..., errors="warn")` exist? I don't see it.. https://docs.python.org/3.4/library/codecs.html#error-handlers